docs/writing-plugins.md

```diff
@@ -303,65 +303,65 @@ the event, for example the following could be defined in `info.js`:
     },
 
 This specifies an event that can be sent from the master to a slave,
-and from a slave to an instance.  And that the event must contain the
-property `frobnication_type` with a string value in the data payload.
+and from a slave to an instance.  It also specifies that the event must contain the
+property `frobnication_type`, with a string value in the data payload.
 It will also be forwarded by slaves to a specific instance.
 
 The following properties are recognized by the Event constructor:
 
 **type**:
-    The message type sent over the wire.  This can be any string but
-    it must be unique across all plugins and it's recommended that it is
+    The message type sent over the wire.  This can be any string, but
+    it must be unique across all plugins, and it's recommended that it is
     of format `plugin_name:message_name`.  The suffix `_event` will be
-    appened to the type.
+    appended to the type.
 
 **links**:
     An array of strings describing which links this event can be sent
     over.  Direction matters, `'master-slave'` means the event can be
-    sent from the master to slave, but can't be sent back the other way
+    sent from the master to the slave, but can't be sent back the other way,
     unless `'slave-master'` is also present in the links array.
 
 **forwardTo**:
-    Target to forward event to.  Can be either `'master'` to indicate a
-    slave should forward it to the master server or `'instance'` to
+    Target to forward an event to.  Can either be `'master'`, to indicate a
+    slave should forward it to the master server, or `'instance'`, to
     indicate it should be forwarded to the instances specified by the
     `instance_id` event property.  This works by using a default handler
     for the event at the links that forward it.
 
 **broadcastTo**:
-    Target to broadcast this message towards.  Currently only
+    Target to broadcast this message towards.  Currently, only
     `'instance'` is supported and means the event will be broadcast to
     all instances downstream of the target it's sent to, but not back
     from where it came from.  This means that sending an event to a
     slave from an instance will cause it to be broadcast to all
-    instances of that slave except for the instance it came from.
+    instances of that slave, except for the instance it came from.
```

Comment:
The changes in #288 should be added into this PR, otherwise it'll just turn into a merge conflict.  You can add changes to this PR by adding commits to your [patch-2 branch](url=https://github.com/MyNameIsTrez/factorioClusterio/tree/patch-2).